docs/getting_started.md

```diff
@@ -8,83 +8,103 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.8
 kernelspec:
-  display_name: 'Python 3.8.11 (''.venv'': venv)'
+  display_name: Python 3
   language: python
   name: python3
 ---
 
-+++ {"id": "9q9JZjt16AMf"}
++++ {"id": "6eea21b3"}
 
 [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/flax/blob/main/docs/getting_started.ipynb)
 [![Open On GitHub](https://img.shields.io/badge/Open-on%20GitHub-blue?logo=GitHub)](https://github.com/google/flax/blob/main/docs/getting_started.ipynb)
 
-# Getting Started
+# Quick Start
 
-This tutorial demonstrates how to construct a simple convolutional neural
+Welcome to Flax!
+
+Flax is an open source Python neural network library built on top of [JAX](https://github.com/google/jax). This tutorial demonstrates how to construct a simple convolutional neural
 network (CNN) using the [Flax](https://flax.readthedocs.io) Linen API and train
 the network for image classification on the MNIST dataset.
 
-Note: This notebook is based on Flax's official
-[MNIST Example](https://github.com/google/flax/tree/main/examples/mnist).
-If you see any changes between the two feel free to create a
-[pull request](https://github.com/google/flax/compare)
-to synchronize this Colab with the actual example.
-
-```{code-cell}
----
-colab:
-  base_uri: https://localhost:8080/
-id: FOYfP6vf2FGx
-outputId: 65ba547d-59df-4fc0-9578-9faa2c14abe8
-tags: [skip-execution]
----
-# Check GPU
-!nvidia-smi -L
-```
-
-+++ {"id": "VSaA8Mif6srP"}
++++ {"id": "nwJWKIhdwxDo"}
 
-## 1. Imports
-
-Import JAX, [JAX NumPy](https://jax.readthedocs.io/en/latest/jax.numpy.html),
-Flax, ordinary NumPy, and TensorFlow Datasets (TFDS). Flax can use any
-data-loading pipeline and this example demonstrates how to utilize TFDS.
+## 1. Install Flax
 
 ```{code-cell}
-:id: inJ9bV636dRx
+:id: bb81587e
 :tags: [skip-execution]
 
 !pip install -q flax
 ```
 
++++ {"id": "b529fbef"}
+
+## 2. Loading data
+
+Flax can use any
+data-loading pipeline and this example demonstrates how to utilize TFDS. Define a function that loads and prepares the MNIST dataset and converts the
```

Comment:
Nit:

```suggestion
data-loading pipeline and this example demonstrates how to utilize TensorFlow Datasets (TFDS). Define a function that loads and prepares the MNIST dataset and converts the
```

Since we haven't mentioned TFDS before, it may help to spell out the full name of the library